Assets/_Scripts/Overworld/Events/CampingGrounds.cs

```diff
@@ -0,0 +1,11 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
```

Comment:
Doesn't look like there is an event for Battle

Reply:
Events will be used for non-battle stages. I plan to have battles processed another way (for now, it just sends the player to the Battle scene)

Reply:
Did you want to split that into another task? It makes sense that processing battle data should be in the same PR, but it might require refactor on existing stage logic. Your choice.

Reply:
Yup, I'm working on it now as a separate PR. I'll split the task up accordingly

Reply:
nvm there's already a card